CHANGELOG.md

```diff
@@ -28,6 +28,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
               locations when running the `release` target of the driver's root
               Makefile.
 * [`fixed`]   Fix `cpp` build of SVM30 due to missing header file
+* [`fixed`]   SVM30: Fix calculation of absolute humidity at very low
+              temperatures (< -20°C / -4°F). The conversion now bounds the
+              result to the lowest result from the look-up table.
+              Also become tolerant towards accepting negative %RH values.
+* [`fixed`]   SVM30: Fix disabling of humidity compensation at values < 0.08%RH
```

Comment:
Isn't about the absolute humidity?

Reply:
At this low RH, the RH+T to AH conversion will always return 0 due to the first interpolation point being above that threshold.

Reply:
OK.